crates/networking/rpc/utils.rs

```diff
@@ -321,7 +322,7 @@ pub mod test_utils {
         start_api(
             http_addr,
             authrpc_addr,
-            storage,
+            blockchain,
```

Comment:
let's pass both `storage` and `blockchain` since these are different entities and shouldn't be coupled together.